package.json

```diff
@@ -35,17 +35,6 @@
         "build": "grunt",
         "test": "grunt test"
     },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "lint-staged": {
-        "*.js": [
-            "eslint --fix",
-            "git add"
-        ]
-    },
```

Comment:
Where is `lint-staged` going?

Reply:
According to [husky 5 installation guide](https://typicode.github.io/husky/#/?id=add-a-hook) it goes to https://github.com/PastVu/pastvu/blob/master/.husky/pre-commit.